usage-updater/index.js

```diff
@@ -0,0 +1,21 @@
+const Rx = require('rx');
+const Notifications = require('./lib/Notifications');
+const Lambda        = require('./lib/Lambda');
+const UsageRequest  = require('./lib/UsageRequest');
+
+
+exports.handler = function(event, context) {
```

Comment:
I think we safely use the "free" Lambda invocation/error statistics to monitor this service if necessary. Usage updates are already recorded via the `usage` app itself.